CHANGELOG.md

```diff
@@ -1,272 +1,154 @@
 # Changelog
 
-## [6.7.0](https://github.com/voxpupuli/beaker/tree/6.6.0) (2025-05-28)
```

Comment:
github fails to render the diff properly. It looks fine locally. With the gcg fix from @smortex , we can finally generate the CHANGELOG.md again. For the past 2 years, we wrote it by hand.